lib/recaptcha/reply.rb

```diff
@@ -0,0 +1,116 @@
+module Recaptcha
+  class Reply
+    def initialize(raw_reply, enterprise: false)
+      @raw_reply = raw_reply
+      @enterprise = enterprise
+    end
+
+    def token_properties
+      @raw_reply['tokenProperties'] if enterprise?
+    end
+
+    def success?(options = {})
```

Comment:
having a `?` return a non-bool is a bit sus
```suggestion
    def success(options = {})
```

Reply:
right, in that case we can keep `success?` like a boolean for test `success` and add `validate_success` method. What do you think?